erpnext/projects/doctype/task/task.json

```diff
@@ -133,7 +133,7 @@
    "no_copy": 1,
    "oldfieldname": "status",
    "oldfieldtype": "Select",
-   "options": "Open\nWorking\nPending Review\nOverdue\nCompleted\nCancelled"
+   "options": "Open\nWorking\nPending Review\nOn Hold\nOverdue\nCompleted\nCancelled"
```

Comment:
Lets remove Overdue Status options, and instead add an "Overdue" checkbox field. Update background worker method for setting overdue, check all references where Overdue status is being set or used. Add patch to set status for Overdue to Working tasks if any

Reply:
- Read only, no copy checkbox `is_overdue`
- Change `Task.update_status()` to `Task.set_is_overdue(update=False, update_modifed=False)`
- Move all over due determination logic in the `Task.set_is_overdue()` method
- Call `Task.set_is_overdue()` in validate